src/index.html

```diff
@@ -139,6 +139,11 @@
         margin: 0.75em 0 0.75em 0.5em;
         height: 1.5em;
       }
+      dd {
+        margin-inline-start: 0;
+        margin-bottom: 12px;
+        margin-left: 0 !important;
```

Comment:
@danid123 is the important here good? I have tried to stay away from that. 

Aside from this question it looks good to me.

Reply:
I had to add that because the WCAG styling was overriding our changes.

Reply:
Damn..  Ok.  I think I can mark this as good then.